FogCreek/Glitch-Community
# Outstanding pamphlet: finish componentizing ProjectsList

## Links
* Remix link: https://outstanding-pamphlet.glitch.me
* Manuscript link: https://glitch.manuscript.com/f/cases/3328436/

## GIF/Screenshots
Home - Recent Projects https://outstanding-pamphlet.glitch.me
![Screen Shot 2019-04-30 at 4 10 57 PM](https://user-images.githubusercontent.com/938722/56990584-82afbb80-6b63-11e9-8c21-3ac9bf064f66.png)

Home - Featured Collections https://outstanding-pamphlet.glitch.me
![Screen Shot 2019-04-30 at 4 11 04 PM](https://user-images.githubusercontent.com/938722/56990585-82afbb80-6b63-11e9-9cdc-333e33446d16.png)

User - Pinned & Recent projects https://outstanding-pamphlet.glitch.me/@modernserf
![Screen Shot 2019-04-30 at 4 11 21 PM](https://user-images.githubusercontent.com/938722/56990586-82afbb80-6b63-11e9-839c-63122363ade8.png)

Team - Pinned & Recent projects https://outstanding-pamphlet.glitch.me/@glitch
![Screen Shot 2019-04-30 at 4 11 40 PM](https://user-images.githubusercontent.com/938722/56990587-82afbb80-6b63-11e9-8c05-5b2658454800.png)

Project - More By authors https://outstanding-pamphlet.glitch.me/~community
![Screen Shot 2019-04-30 at 4 12 18 PM](https://user-images.githubusercontent.com/938722/56990588-82afbb80-6b63-11e9-9385-1a8b5c2ff99e.png)

Collection https://outstanding-pamphlet.glitch.me/@glitch/featured-on-glitch-march-29-2019
![Screen Shot 2019-04-30 at 4 15 37 PM](https://user-images.githubusercontent.com/938722/56990589-82afbb80-6b63-11e9-812e-978e1eaa37fc.png)

Category https://outstanding-pamphlet.glitch.me/hello-worlds
![Screen Shot 2019-04-30 at 4 15 56 PM](https://user-images.githubusercontent.com/938722/56990590-82afbb80-6b63-11e9-9c09-f0e16d8afcac.png)

## Changes:
* modularize basic ProjectList styles
* pass in extra class name to configure style variations (e.g. different breakpoints/min-widths for grid)
* handle the margins/padding in the parent containers
* delete unused CSS

## How To Test:
* visit the pages linked above and verify that the project lists look right at various viewport sizes
